### DISCUSSION TITLE
buddhism/latest/todo/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#116] [General]
# buddhism/latest/todo/

https://shanenull.com/buddhism/latest/todo/

**Comment by shane0** [comment]
# new experiments

- first principles
- potato chip effect
- troubleshooting template log
- first letters primordial confusions

**Comment by shane0** [comment]
- [ ] add button to this on the homepage [mind ground](https://shanenull.com/buddhism/latest/mind_ground/)
